### PR TITLE
fix: Fix MerkleDistributor unit tests

### PR DESCRIPTION
**Pull request by nicholaspai**
Current tests import MerkleTree from @uma/merkle-distrbutor which is a deprecated package. This PR imports MerkleTree from @uma/common and uses the latest MerkleTree class

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,5 @@
 [submodule "packages/core/temp/lib/forge-std"]
 	path = packages/core/temp/lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
+[submodule "lib/forge-std"]
```
Comment (nicholaspai) [inline .gitmodules:4]: I ran `yarn clean` and `yarn qbuild` and this happened. Should it be here?

Comment (mrice32) [reply .gitmodules:4]: @chrismaree is this expected? Or should we drop this from the diff?

**Review by mrice32 — APPROVED** [review]
Other than the odd .gitmodules change, this LGTM!

File: .gitmodules (hunk 1 above)
Comment (chrismaree) [reply .gitmodules:4]: I think it should? im not sure why it was not pulled in previously but we use this via submodules.

**Pull request merged**